package.json

```diff
@@ -6,7 +6,7 @@
   "author": "The Gaia Team <dev-gaia@lists.mozilla.org>",
   "engineStrict": "true",
   "engines": {
-    "node": ">=0.10",
+    "node": ">=4.0",
```

Comment:
But to change the engines along with sockit, shouldn't Gaia also version change? Idk, just wondering. :)